metacatalog/ext/standards_export/extension.py

```diff
@@ -35,6 +37,26 @@
         organisation_name = 'METACATALOG'
         ))
 
+# default v-for-water contact, TODO: store somewhere else (needed for cli)
+V4W_CONTACT_DICT = {
+    "contact": {
+        "organisationName": "Karlsruhe Institute of Technology (KIT) - Institute of Water and River Basin Management - Chair of Hydrology",
+        "deliveryPoint": "Otto-Ammann-Platz 1, Building 10.81",
+        "city": "Karlsruhe",
+        "administrativeArea": "Baden-Wuerttemberg",
+        "postalCode": "76131",
+        "country": "Germany",
+        "electronicMailAddress": ["alexander.dolich@kit.edu", "mirko.maelicke@kit.edu"],
+        "linkage": "https://portal.vforwater.de/",
+        "linkage_name": "V-FOR-WaTer",
+        "linkage_description": "Virtual research environment for water and terrestrial environmental research"
+    },
+    "publisher": {
+        "organisation_name": "KIT, V-For-WaTer online platform"
+    }
+}
+
```

Comment:
I still think this is a really bad idea. We should not hardcode our personal contact info into the package. The extension should be able to handle empty dictionaries, just like we discussed.